stdlib/ballerina-builtin/src/main/ballerina/ballerina/file/natives.bal

```diff
@@ -59,33 +62,32 @@ public struct FileNotOpenedError {
     StackFrame[] stackTrace;
 }
 
-@Description { value:"Closes a given file and its stream"}
-@Param { value: "file: The file to be closed"}
-public native function <File file> close ();
-
-@Description { value:"Retrieves the stream from a local file"}
-@Param { value:"file: The file which needs to be opened" }
-@Param { value:"accessMode: The file access mode used when opening the file" }
-public native function <File file> open (string accessMode);
-
 @Description { value:"Copies a file from a given location to another"}
 @Param { value:"source: File/Directory that should be copied" }
 @Param { value:"destination: Destination directory or path to which the source should be copied" }
```

Comment:
Shall we reformat properly- Extra space after the "{". Better if can reformat the whole file